display/d.linegraph/main.c

```diff
@@ -889,16 +890,16 @@ int main(int argc, char **argv)
 
     /* draw the x-axis label */
     if ((strcmp(title[0]->answer, "") == 0) && (strcmp(tic_name, "") == 0))
-        *xlabel = '\0';
+        *pxlabel = '\0';
```

Comment:
```suggestion
        xlabel = G_store("");
```

Reply:
Yes, of course you're right, I completely overlooked the need for a dynamic memory allocation.